tools/wptrunner/wptrunner/formatters.py

```diff
@@ -8,15 +8,21 @@ class WptreportFormatter(BaseFormatter):
 
     def __init__(self):
         self.raw_results = {}
+        self.results = {}
+
+    def suite_start(self, data):
+        self.results['browser'] = data['run_info']['product']
```

Comment:
I would be tempted to just dump `run_info` directly into the results.

Reply:
Yeah let's do that.

Meanwhile, could you answer the question I had regarding these fields in `run_info`?
```
"linux_distro": "Debian",
"os": "linux",
"os_version": "sid",
"version": "Debian sid",
```
They seem to present an overlapping set of information. Is this intended? Notably, what's `version`? Is it intended to be the version of the product or the OS?

Reply:
All I know is what I can figure out from `https://searchfox.org/mozilla-central/source/testing/mozbase/mozinfo/mozinfo/mozinfo.py`. So `version` is what you get from `uname` and `os_version` has a more custom  implementation.